traj_topic:   /trajectory

update_rate:  0.1
